composer/models/huggingface.py

```diff
@@ -70,12 +70,11 @@ def __init__(self,
         self.use_logits = use_logits
 
         self.train_metrics = None
-        self.valid_metrics = None
+        self.val_metrics = None
 
         if metrics:
-            metric_collection = MetricCollection(metrics)
-            self.train_metrics = metric_collection.clone(prefix='train_')
-            self.valid_metrics = metric_collection.clone(prefix='val_')
+            self.train_metrics = {metric._get_name(): metric for metric in metrics}
```

Comment:
We shouldn't rely on a private method from torchmetrics here.. can this just be the class name?